### PR TITLE
Change hover to build without Docker

### PR DESCRIPTION
**Pull request by thiagokokada**
Trying to build Hover without Docker.

### CODE REVIEW

File: pkgs/hover-nix/default.nix [hunk 1]
```diff
@@ -56,6 +56,8 @@ in buildGoModule rec {
     chmod -R a+rx $out/share/assets
 
     wrapProgram "$out/bin/hover" \
+      --prefix PATH : ${stdenv.lib.makeBinPath [ gcc ]} \
+      --prefix CPATH : ${xorg.libX11}/share:${xorg.libXcursor}/share:${xorg.libXi}/share:${xorg.libXinerama}/share:${xorg.libXrandr}/share:${xorg.libXxf86vm}/share:${xorg.libXext}/share \
```
Comment (ericdallo) [inline pkgs/hover-nix/default.nix]:
```suggestion
      --prefix CPATH : ${xorg.libX11.dev}/include/X11:${xorg.libXcursor.dev}/include/X11/Xcursor:${xorg.libXi.dev}/include/X11/extensions:${xorg.libXinerama.dev}/include/X11/extensions:${xorg.libXrandr.dev}/include/X11/extensions:${xorg.libXxf86vm.dev}/include/X11/extensions:${xorg.libXext.dev}/include/X11/extensions \
```

File: default.nix [hunk 2]
```diff
@@ -5,7 +5,7 @@ in {
   nubank = {
     dart = callPackage ./pkgs/dart {};
     flutter = callPackage ./pkgs/flutter {};
-    hover = callPackage ./pkgs/hover {
+    hover = callPackage ./pkgs/hover-nix {
```
Comment (ericdallo) [inline default.nix]: We can renamed it to `hover` only if now works, right?
```suggestion
    hover = callPackage ./pkgs/hover {
```

Comment (thiagokokada) [reply default.nix]: Done, sorry I forgot to remove it.

**Comment by thiagokokada** [comment]
Runing with `nix-shell --pure`:

```shell
[nix-shell:~/dev/nu/mini-meta-repo]$ hover run
hover: Missing config: file hover.yaml not found: open go/hover.yaml: no such file or directory
hover: Failed to lookup `flutter` executable: exec: "flutter": executable file not found in $PATH. Please install flutter or add `--docker` to run the Hover command in Docker container.
https://flutter.dev/docs/get-started/install
```

So we probably need to pass `flutter` as the `buildInput` also. But I think I will do this in another PR.

File: pkgs/hover/default.nix [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
+    description = "A build tool to run Flutter applications on desktop";
+    homepage = "https://github.com/go-flutter-desktop/hover";
+    license = licenses.bsd3;
+    platforms = platforms.linux ++ platforms.darwin;
+  };
+
   subPackages = [ "." ];
 
-  vendorSha256 = "1wr08phjm87dxim47i8449rmq5wfscvjyz65g3lxmv468x209pam";
+  modSha256 = "0qg26bzbdmb0cl2msgg2ycxhdkhiqbriihq9f725w7a6j0mcbz3a";
```
Comment (ericdallo) [inline pkgs/hover/default.nix]:
```suggestion
  vendorSha256 = "1wr08phjm87dxim47i8449rmq5wfscvjyz65g3lxmv468x209pam";
  modSha256 = "0qg26bzbdmb0cl2msgg2ycxhdkhiqbriihq9f725w7a6j0mcbz3a"; # Remove after NixOS 20.09
```

**Review by ericdallo — APPROVED** [review]

**Pull request merged**